kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wdt/create-domain-script.sh

```diff
@@ -60,6 +60,8 @@
 #                  $WDT_MODEL_FILE and $WDT_MODEL_VARS. Also the location
 #                  of WDT log files.
 #                  default:  /shared/wdt
+#   WDT_VERSION     WDT version to download.
```

Comment:
i think you have one extra space before "WDT version...."

Reply:
Removed the extra space. Thanks.